source/geometry/IntPoint2D.ooc

```diff
@@ -23,6 +23,7 @@ IntPoint2D: cover {
 	minimum: func (ceiling: This) -> This { This new(this x minimum(ceiling x), this y minimum(ceiling y)) }
 	maximum: func (floor: This) -> This { This new(this x maximum(floor x), this y maximum(floor y)) }
 	clamp: func (floor, ceiling: This) -> This { This new(this x clamp(floor x, ceiling x), this y clamp(floor y, ceiling y)) }
+	mix: static func (a, b: This, ratio: Float) -> This { FloatPoint2D mix(a toFloatPoint2D(), b toFloatPoint2D(), ratio) round() toIntPoint2D() }
```

Comment:
Also, sorry to be late on this comment, but since this is a static function, put it just above the other static function `parse`. Then we're done here.